{"pagelen": 100, "values": [{"changes": {"content": {"new": "```\r\n#!bash\r\n\r\n['sed', '-i', '-e', 's/\\\\(\\\\.tga\\\\|\\\\.tiff\\\\|\\\\.tif\\\\|\\\\.jpg\\\\|\\\\.jpeg\\\\|\\\\.gif\\\\)/\\\\.png/g', 'Male.dae']\r\nsed: can't read Male.dae: No such file or directory\r\nerror Command '['sed', '-i', '-e', 's/\\\\(\\\\.tga\\\\|\\\\.tiff\\\\|\\\\.tif\\\\|\\\\.jpg\\\\|\\\\.jpeg\\\\|\\\\.gif\\\\)/\\\\.png/g', 'Male.dae']' returned non-zero exit status 2\r\nTraceback (most recent call last):\r\n  File \"./webify_models_v2.py\", line 67, in <module>\r\n    subprocess.check_call(sed_cmd)\r\n  File \"/usr/lib/python2.7/subprocess.py\", line 190, in check_call\r\n    raise CalledProcessError(retcode, cmd)\r\nsubprocess.CalledProcessError: Command '['sed', '-i', '-e', 's/\\\\(\\\\.tga\\\\|\\\\.tiff\\\\|\\\\.tif\\\\|\\\\.jpg\\\\|\\\\.jpeg\\\\|\\\\.gif\\\\)/\\\\.png/g', 'Male.dae']' returned non-zero exit status 2\r\n\r\n```\r\n\r\nAdded a model that is named `Large Male`\r\n\r\n\r\n```\r\n#!bash\r\n'Survivor Male'\r\n\u251c\u2500\u2500 materials\r\n\u2502\u00a0\u00a0 \u2514\u2500\u2500 textures\r\n\u2502\u00a0\u00a0     \u2514\u2500\u2500 Survivor Male.png\r\n\u251c\u2500\u2500 meshes\r\n\u2502\u00a0\u00a0 \u2514\u2500\u2500 Survivor Male.dae\r\n\u251c\u2500\u2500 model.config\r\n\u2514\u2500\u2500 model.sdf\r\n```\r\n\r\ngazebo 9.4.1\r\nGzweb on branch insert_sdf\r\nissue #158\r\n#!/osrf/gzweb/pull-requests/173", "old": "```\r\n#!bash\r\n\r\n['sed', '-i', '-e', 's/\\\\(\\\\.tga\\\\|\\\\.tiff\\\\|\\\\.tif\\\\|\\\\.jpg\\\\|\\\\.jpeg\\\\|\\\\.gif\\\\)/\\\\.png/g', 'Male.dae']\r\nsed: can't read Male.dae: No such file or directory\r\nerror Command '['sed', '-i', '-e', 's/\\\\(\\\\.tga\\\\|\\\\.tiff\\\\|\\\\.tif\\\\|\\\\.jpg\\\\|\\\\.jpeg\\\\|\\\\.gif\\\\)/\\\\.png/g', 'Male.dae']' returned non-zero exit status 2\r\nTraceback (most recent call last):\r\n  File \"./webify_models_v2.py\", line 67, in <module>\r\n    subprocess.check_call(sed_cmd)\r\n  File \"/usr/lib/python2.7/subprocess.py\", line 190, in check_call\r\n    raise CalledProcessError(retcode, cmd)\r\nsubprocess.CalledProcessError: Command '['sed', '-i', '-e', 's/\\\\(\\\\.tga\\\\|\\\\.tiff\\\\|\\\\.tif\\\\|\\\\.jpg\\\\|\\\\.jpeg\\\\|\\\\.gif\\\\)/\\\\.png/g', 'Male.dae']' returned non-zero exit status 2\r\n\r\n```\r\n\r\nAdded a model that is named `Large Male`\r\n\r\n\r\n```\r\n#!bash\r\n'Survivor Male'\r\n\u251c\u2500\u2500 materials\r\n\u2502\u00a0\u00a0 \u2514\u2500\u2500 textures\r\n\u2502\u00a0\u00a0     \u2514\u2500\u2500 Survivor Male.png\r\n\u251c\u2500\u2500 meshes\r\n\u2502\u00a0\u00a0 \u2514\u2500\u2500 Survivor Male.dae\r\n\u251c\u2500\u2500 model.config\r\n\u2514\u2500\u2500 model.sdf\r\n```\r\n\r\ngazebo 9.4.1\r\nGzweb on branch insert_sdf\r\nissue #173\r\n#!/osrf/gzweb/pull-requests/173"}}, "links": {"self": {"href": "data/repositories/osrf/gzweb/issues/160/changes/49015810.json"}, "html": {"href": "#!/osrf/gzweb/issues/160#comment-49015810"}}, "issue": {"links": {"self": {"href": "data/repositories/osrf/gzweb/issues/160.json"}}, "type": "issue", "id": 160, "repository": {"links": {"self": {"href": "data/repositories/osrf/gzweb.json"}, "html": {"href": "#!/osrf/gzweb"}, "avatar": {"href": "data/bytebucket.org/ravatar/{3e2fc3f2-dc25-453b-9792-943c743d9b40}ts=1609263"}}, "type": "repository", "name": "gzweb", "full_name": "osrf/gzweb", "uuid": "{3e2fc3f2-dc25-453b-9792-943c743d9b40}"}, "title": "deploy.sh does not handle spaces in model filenames"}, "created_on": "2018-11-19T19:55:47.177459+00:00", "user": {"display_name": "Will Baker", "uuid": "{29059f61-e8ef-4122-a6c9-ebf4207e716a}", "links": {"self": {"href": "https://api.bitbucket.org/2.0/users/%7B29059f61-e8ef-4122-a6c9-ebf4207e716a%7D"}, "html": {"href": "https://bitbucket.org/%7B29059f61-e8ef-4122-a6c9-ebf4207e716a%7D/"}, "avatar": {"href": "data/secure.gravatar.com/avatar/1ef1c401594173d16b47bc3b10e3dc75d=httpsavatar-management--avatars.us-west-2.prod.public.atl-paas.netinitialsWB-5.png"}}, "nickname": "William Baker", "type": "user", "account_id": "557058:8aa0b9c9-7cec-46ea-9d87-86b91c48c7a6"}, "message": {"raw": null, "markup": "markdown", "html": "", "type": "rendered"}, "type": "issue_change", "id": 49015810}, {"changes": {"content": {"new": "```\r\n#!bash\r\n\r\n['sed', '-i', '-e', 's/\\\\(\\\\.tga\\\\|\\\\.tiff\\\\|\\\\.tif\\\\|\\\\.jpg\\\\|\\\\.jpeg\\\\|\\\\.gif\\\\)/\\\\.png/g', 'Male.dae']\r\nsed: can't read Male.dae: No such file or directory\r\nerror Command '['sed', '-i', '-e', 's/\\\\(\\\\.tga\\\\|\\\\.tiff\\\\|\\\\.tif\\\\|\\\\.jpg\\\\|\\\\.jpeg\\\\|\\\\.gif\\\\)/\\\\.png/g', 'Male.dae']' returned non-zero exit status 2\r\nTraceback (most recent call last):\r\n  File \"./webify_models_v2.py\", line 67, in <module>\r\n    subprocess.check_call(sed_cmd)\r\n  File \"/usr/lib/python2.7/subprocess.py\", line 190, in check_call\r\n    raise CalledProcessError(retcode, cmd)\r\nsubprocess.CalledProcessError: Command '['sed', '-i', '-e', 's/\\\\(\\\\.tga\\\\|\\\\.tiff\\\\|\\\\.tif\\\\|\\\\.jpg\\\\|\\\\.jpeg\\\\|\\\\.gif\\\\)/\\\\.png/g', 'Male.dae']' returned non-zero exit status 2\r\n\r\n```\r\n\r\nAdded a model that is named `Large Male`\r\n\r\n\r\n```\r\n#!bash\r\n'Survivor Male'\r\n\u251c\u2500\u2500 materials\r\n\u2502\u00a0\u00a0 \u2514\u2500\u2500 textures\r\n\u2502\u00a0\u00a0     \u2514\u2500\u2500 Survivor Male.png\r\n\u251c\u2500\u2500 meshes\r\n\u2502\u00a0\u00a0 \u2514\u2500\u2500 Survivor Male.dae\r\n\u251c\u2500\u2500 model.config\r\n\u2514\u2500\u2500 model.sdf\r\n```\r\n\r\ngazebo 9.4.1\r\nGzweb on branch insert_sdf\r\nusing docker file from issue #158\r\nand pull request #173\r\n#!/osrf/gzweb/pull-requests/173", "old": "```\r\n#!bash\r\n\r\n['sed', '-i', '-e', 's/\\\\(\\\\.tga\\\\|\\\\.tiff\\\\|\\\\.tif\\\\|\\\\.jpg\\\\|\\\\.jpeg\\\\|\\\\.gif\\\\)/\\\\.png/g', 'Male.dae']\r\nsed: can't read Male.dae: No such file or directory\r\nerror Command '['sed', '-i', '-e', 's/\\\\(\\\\.tga\\\\|\\\\.tiff\\\\|\\\\.tif\\\\|\\\\.jpg\\\\|\\\\.jpeg\\\\|\\\\.gif\\\\)/\\\\.png/g', 'Male.dae']' returned non-zero exit status 2\r\nTraceback (most recent call last):\r\n  File \"./webify_models_v2.py\", line 67, in <module>\r\n    subprocess.check_call(sed_cmd)\r\n  File \"/usr/lib/python2.7/subprocess.py\", line 190, in check_call\r\n    raise CalledProcessError(retcode, cmd)\r\nsubprocess.CalledProcessError: Command '['sed', '-i', '-e', 's/\\\\(\\\\.tga\\\\|\\\\.tiff\\\\|\\\\.tif\\\\|\\\\.jpg\\\\|\\\\.jpeg\\\\|\\\\.gif\\\\)/\\\\.png/g', 'Male.dae']' returned non-zero exit status 2\r\n\r\n```\r\n\r\nAdded a model that is named `Large Male`\r\n\r\n\r\n```\r\n#!bash\r\n'Survivor Male'\r\n\u251c\u2500\u2500 materials\r\n\u2502\u00a0\u00a0 \u2514\u2500\u2500 textures\r\n\u2502\u00a0\u00a0     \u2514\u2500\u2500 Survivor Male.png\r\n\u251c\u2500\u2500 meshes\r\n\u2502\u00a0\u00a0 \u2514\u2500\u2500 Survivor Male.dae\r\n\u251c\u2500\u2500 model.config\r\n\u2514\u2500\u2500 model.sdf\r\n```\r\n\r\ngazebo 9.4.1\r\nGzweb on branch insert_sdf\r\nissue #158\r\n#!/osrf/gzweb/pull-requests/173"}}, "links": {"self": {"href": "data/repositories/osrf/gzweb/issues/160/changes/49015819.json"}, "html": {"href": "#!/osrf/gzweb/issues/160#comment-49015819"}}, "issue": {"links": {"self": {"href": "data/repositories/osrf/gzweb/issues/160.json"}}, "type": "issue", "id": 160, "repository": {"links": {"self": {"href": "data/repositories/osrf/gzweb.json"}, "html": {"href": "#!/osrf/gzweb"}, "avatar": {"href": "data/bytebucket.org/ravatar/{3e2fc3f2-dc25-453b-9792-943c743d9b40}ts=1609263"}}, "type": "repository", "name": "gzweb", "full_name": "osrf/gzweb", "uuid": "{3e2fc3f2-dc25-453b-9792-943c743d9b40}"}, "title": "deploy.sh does not handle spaces in model filenames"}, "created_on": "2018-11-19T19:56:39.573666+00:00", "user": {"display_name": "Will Baker", "uuid": "{29059f61-e8ef-4122-a6c9-ebf4207e716a}", "links": {"self": {"href": "https://api.bitbucket.org/2.0/users/%7B29059f61-e8ef-4122-a6c9-ebf4207e716a%7D"}, "html": {"href": "https://bitbucket.org/%7B29059f61-e8ef-4122-a6c9-ebf4207e716a%7D/"}, "avatar": {"href": "data/secure.gravatar.com/avatar/1ef1c401594173d16b47bc3b10e3dc75d=httpsavatar-management--avatars.us-west-2.prod.public.atl-paas.netinitialsWB-5.png"}}, "nickname": "William Baker", "type": "user", "account_id": "557058:8aa0b9c9-7cec-46ea-9d87-86b91c48c7a6"}, "message": {"raw": null, "markup": "markdown", "html": "", "type": "rendered"}, "type": "issue_change", "id": 49015819}, {"changes": {"content": {"new": "```\r\n#!bash\r\n\r\n['sed', '-i', '-e', 's/\\\\(\\\\.tga\\\\|\\\\.tiff\\\\|\\\\.tif\\\\|\\\\.jpg\\\\|\\\\.jpeg\\\\|\\\\.gif\\\\)/\\\\.png/g', 'Male.dae']\r\nsed: can't read Male.dae: No such file or directory\r\nerror Command '['sed', '-i', '-e', 's/\\\\(\\\\.tga\\\\|\\\\.tiff\\\\|\\\\.tif\\\\|\\\\.jpg\\\\|\\\\.jpeg\\\\|\\\\.gif\\\\)/\\\\.png/g', 'Male.dae']' returned non-zero exit status 2\r\nTraceback (most recent call last):\r\n  File \"./webify_models_v2.py\", line 67, in <module>\r\n    subprocess.check_call(sed_cmd)\r\n  File \"/usr/lib/python2.7/subprocess.py\", line 190, in check_call\r\n    raise CalledProcessError(retcode, cmd)\r\nsubprocess.CalledProcessError: Command '['sed', '-i', '-e', 's/\\\\(\\\\.tga\\\\|\\\\.tiff\\\\|\\\\.tif\\\\|\\\\.jpg\\\\|\\\\.jpeg\\\\|\\\\.gif\\\\)/\\\\.png/g', 'Male.dae']' returned non-zero exit status 2\r\n\r\n```\r\n\r\nAdded a model that is named `Large Male`\r\n\r\n\r\n```\r\n#!bash\r\n'Large Male'\r\n\u251c\u2500\u2500 materials\r\n\u2502\u00a0\u00a0 \u2514\u2500\u2500 textures\r\n\u2502\u00a0\u00a0     \u2514\u2500\u2500 Large Male.png\r\n\u251c\u2500\u2500 meshes\r\n\u2502\u00a0\u00a0 \u2514\u2500\u2500 Large Male.dae\r\n\u251c\u2500\u2500 model.config\r\n\u2514\u2500\u2500 model.sdf\r\n```\r\n\r\ngazebo 9.4.1\r\nGzweb on branch insert_sdf\r\nusing docker file from issue #158\r\nand pull request #173\r\n#!/osrf/gzweb/pull-requests/173", "old": "```\r\n#!bash\r\n\r\n['sed', '-i', '-e', 's/\\\\(\\\\.tga\\\\|\\\\.tiff\\\\|\\\\.tif\\\\|\\\\.jpg\\\\|\\\\.jpeg\\\\|\\\\.gif\\\\)/\\\\.png/g', 'Male.dae']\r\nsed: can't read Male.dae: No such file or directory\r\nerror Command '['sed', '-i', '-e', 's/\\\\(\\\\.tga\\\\|\\\\.tiff\\\\|\\\\.tif\\\\|\\\\.jpg\\\\|\\\\.jpeg\\\\|\\\\.gif\\\\)/\\\\.png/g', 'Male.dae']' returned non-zero exit status 2\r\nTraceback (most recent call last):\r\n  File \"./webify_models_v2.py\", line 67, in <module>\r\n    subprocess.check_call(sed_cmd)\r\n  File \"/usr/lib/python2.7/subprocess.py\", line 190, in check_call\r\n    raise CalledProcessError(retcode, cmd)\r\nsubprocess.CalledProcessError: Command '['sed', '-i', '-e', 's/\\\\(\\\\.tga\\\\|\\\\.tiff\\\\|\\\\.tif\\\\|\\\\.jpg\\\\|\\\\.jpeg\\\\|\\\\.gif\\\\)/\\\\.png/g', 'Male.dae']' returned non-zero exit status 2\r\n\r\n```\r\n\r\nAdded a model that is named `Large Male`\r\n\r\n\r\n```\r\n#!bash\r\n'Survivor Male'\r\n\u251c\u2500\u2500 materials\r\n\u2502\u00a0\u00a0 \u2514\u2500\u2500 textures\r\n\u2502\u00a0\u00a0     \u2514\u2500\u2500 Survivor Male.png\r\n\u251c\u2500\u2500 meshes\r\n\u2502\u00a0\u00a0 \u2514\u2500\u2500 Survivor Male.dae\r\n\u251c\u2500\u2500 model.config\r\n\u2514\u2500\u2500 model.sdf\r\n```\r\n\r\ngazebo 9.4.1\r\nGzweb on branch insert_sdf\r\nusing docker file from issue #158\r\nand pull request #173\r\n#!/osrf/gzweb/pull-requests/173"}}, "links": {"self": {"href": "data/repositories/osrf/gzweb/issues/160/changes/49015823.json"}, "html": {"href": "#!/osrf/gzweb/issues/160#comment-49015823"}}, "issue": {"links": {"self": {"href": "data/repositories/osrf/gzweb/issues/160.json"}}, "type": "issue", "id": 160, "repository": {"links": {"self": {"href": "data/repositories/osrf/gzweb.json"}, "html": {"href": "#!/osrf/gzweb"}, "avatar": {"href": "data/bytebucket.org/ravatar/{3e2fc3f2-dc25-453b-9792-943c743d9b40}ts=1609263"}}, "type": "repository", "name": "gzweb", "full_name": "osrf/gzweb", "uuid": "{3e2fc3f2-dc25-453b-9792-943c743d9b40}"}, "title": "deploy.sh does not handle spaces in model filenames"}, "created_on": "2018-11-19T19:57:00.640916+00:00", "user": {"display_name": "Will Baker", "uuid": "{29059f61-e8ef-4122-a6c9-ebf4207e716a}", "links": {"self": {"href": "https://api.bitbucket.org/2.0/users/%7B29059f61-e8ef-4122-a6c9-ebf4207e716a%7D"}, "html": {"href": "https://bitbucket.org/%7B29059f61-e8ef-4122-a6c9-ebf4207e716a%7D/"}, "avatar": {"href": "data/secure.gravatar.com/avatar/1ef1c401594173d16b47bc3b10e3dc75d=httpsavatar-management--avatars.us-west-2.prod.public.atl-paas.netinitialsWB-5.png"}}, "nickname": "William Baker", "type": "user", "account_id": "557058:8aa0b9c9-7cec-46ea-9d87-86b91c48c7a6"}, "message": {"raw": null, "markup": "markdown", "html": "", "type": "rendered"}, "type": "issue_change", "id": 49015823}], "page": 1, "size": 3}